plugins/modules/domain.py

```diff
@@ -127,12 +120,9 @@ def run_module():
             msg=f"{exit_message} ! - (dry run mode)"
         )
 
-    try:
-        existing_records = client.get(
-            f"/domain/zone/{domain}/record", fieldType=record_type, subDomain=name
-        )
-    except APIError as api_error:
-        module.fail_json(msg=f"Failed to call OVH API: {api_error}")
+    existing_records = client.prepare_call(
```

Comment:
If this is a "prepare" step, is there is a "commit" step to send everything prepared ? :thinking:

Reply:
Nope, this is just cosmetic. Naming is hard :D